index.html

```diff
@@ -68,24 +76,21 @@ <h1 style="display:inline">CQL Data Modeler</h1>
                 Provide some information about your table definition, and the size, population, and cluster distributions of each of your fields and the tool will give you a script you can use to benchmark your data model and a storage engine visualization.</br></br>
                 Please input your table definition for one table, keeping the Create statement, field definitions, and Primary Key definition on separate lines.</p>
                 <input style="display:block" id="keyspace" placeholder="Keyspace Name" value="test_keyspace"></input>	</br>
-                <textarea id="tableDef" data-role="none"  placeholder="Create Table Statement"  style="width:600px; height:250px">CREATE TABLE reviews_by_day
```

Comment:
no but since it has data in it to start i didnt feel it necessary, ill put it back though in followup here